src/main/kotlin/org/partiql/scribe/sql/SqlDialect.kt

```diff
@@ -69,19 +69,27 @@ abstract class SqlDialect : AstBaseVisitor<SqlBlock, SqlBlock>() {
     // IDENTIFIERS & PATHS
 
     /**
-     * Default behavior is to wrap all SFW queries with parentheses.
+     * Default behavior is to wrap all SFW queries (and binary operators) with parentheses.
```

Comment:
nit: based on my understanding, this is for nested binary operators

```suggestion
     * Default behavior is to wrap all SFW queries (and nested binary operators) with parentheses.
```